frontend/src/components/Overlay/OverlayNav.css

```diff
@@ -0,0 +1,92 @@
+.overlay-nav {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	gap: 1rem;
+	height: 5rem;
+	padding: 0;
+	border-bottom-right-radius: 0.625rem;
+	border-bottom-left-radius: 0.625rem;
+	background-color: var(--overlay-header-background-colour);
+}
+
+.overlay-nav div {
```

Comment:
This is quite a loose selector, is it what you intended - for all divs inside the nav to be centered flex containers?

Looking at the markup, it feels like the two divs wrapping the buttons are unnecessary - can we not just have buttons, assigned to and aligned within their respective grid slots?

Reply:
Since there are only two divs and I wanted to target those two, yes that was intended, but I can see how this could be accidentally targeting more divs if we add more divs, happy to add another class to make it more specific. 

The divs properly center the buttons, without it looks like this: 
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/32af1df3-6f06-4c19-947b-11c3ce43e6a0)
This is working quite well, including with zoom, don't want to fix it if it isn't broken.